include/oxen/quic/ip.hpp

```diff
@@ -21,9 +21,12 @@ namespace oxen::quic
 
         const std::string to_string() const;
 
-        explicit operator const in_addr&() const { return reinterpret_cast<const in_addr&>(addr); }
-
-        explicit operator const uint32_t&() const { return reinterpret_cast<const uint32_t&>(addr); }
+        explicit operator in_addr() const
```

Comment:
Why did you change this to returning a copy? I wanted this conversion operator to use the `ipv4` object as an `in_addr` without copying.

If you need a copy, could we make a `to_inaddr()` function and have the best of both worlds?

Reply:
Makes perfect sense 👍